_data/langs/ru.yml

```diff
@@ -50,87 +50,87 @@ exchanges:
   cryptopia: "Cryptopia"
   coingather: "CoinGather"
 wallets:
-  header: "бумажники"
+  header: "Кошельки"
   tabs:
-    desktop: "рабочий стол"
-    mobile: "мобильный"
+    desktop: "Настольные ОС"
+    mobile: "Мобильные устройства"
     other: "Другие"
-  windows: "Скопировать Windows-Бумажник"
-  macosx: "Вернуть Mac OSX-Бумажник"
-  android: "Вернуть Android-Бумажник"
-  archlinux: "Вершина ArchLinux-Бумажник"
-  centos: "Verge CentOS-Бумажник"
-  debian: "Verge Debian-Бумажник"
-  fedora: "Verge Fedora-Бумажник"
+  windows: "Windows Verge-Кошелёк"
+  macosx: "Mac OSX Verge-Кошелёк"
+  android: "Android Verge-Кошелёк"
+  archlinux: "ArchLinux Verge-Кошелёк"
+  centos: "CentOS Verge-Кошелёк"
+  debian: "Debian Verge-Кошелёк"
+  fedora: "Fedora Verge-Кошелёк"
   web: "Verge Веб-кошелек"
-  web_tor: "Verge TOR Веб-кошелек"
-  web_i2P: "Verge i2P Веб-кошелек"
+  web_tor: "Verge TOR Веб-кошелёк"
+  web_i2P: "Verge i2P Веб-кошелёк"
   raspberry_pi: "Verge Raspberry Pi Исходный код"
-  electrum_linux: "Verge Linux Electrum Бумажник"
-  electrum_linux_tor: "Verge TOR Linux Electrum Бумажник"
-  electrum_windows: "Verge Windows Electrum Бумажник"
-  electrum_windows_tor: "Verge TOR Windows Electrum Бумажник"
-  paper: "Verge Paper Бумажник"
-  daemon_docker: "Verge Daemon Docker Образ"
+  electrum_linux: "Linux Electrum Verge-Кошелёк"
+  electrum_linux_tor: "TOR Linux Electrum Verge-Кошелёк"
+  electrum_windows: "Windows Electrum Verge-Кошелёк"
+  electrum_windows_tor: "TOR Windows Electrum Verge-Кошелёк"
+  paper: "Бумажный Verge-Кошелёк"
+  daemon_docker: "Образ демона Докера для Verge"
   coinomi: "Coinomi Мультивалютный Android-кошелек"
 roadmap:
   header: "Дорожная карта"
   blackpaper:
-    header: "Черная бумага"
-    date: "<font color='red'>ВЫПУЩЕННЫЙ</font> 4 июня 2017 года"
+    header: ""Черная" книга"
```

Comment:
We put `\` in front of a nested double quote. The other option is to use single quotes.